mlir/tools/mlir-tblgen/OpInterfacesGen.cpp

```diff
@@ -23,6 +23,9 @@
 #include "llvm/TableGen/TableGenBackend.h"
 
 using namespace mlir;
+using llvm::interleaveComma;
```

Comment:
I'd drop this one, we generally don't add usings for things like this.

Reply:
Done